packages/lib-webpack/src/webpack/DynamicRemotePlugin.ts

```diff
@@ -0,0 +1,167 @@
+import {
+  extensionArraySchema,
+  remoteEntryScript,
+  remoteEntryCallback,
+} from '@openshift/dynamic-plugin-sdk';
+import type { Extension } from '@openshift/dynamic-plugin-sdk';
+import * as _ from 'lodash-es';
```

Comment:
Yeah, does rollup have tree-shaking? 🤔

Reply:
`lodash-es` is already code that is ESmodules syntax friendly.

All external module imports are reflected into `dist/index.js` as-is. It's the module bundler employed by the consuming plugin or host application that will have to process our SDK code (which is currently expressed as ESmodules).

Reply:
Also, sometimes a namespace import (`import * as foo`) is useful for scoping the imported stuff into a single object.

I agree that it's generally best to pick specific imports from the given library, instead of using a namespace import.

For now, we've been sticking with a namespace import style for vendor modules, let's improve it as a follow-up.